pkg/cli/setup.go

```diff
@@ -0,0 +1,297 @@
+package cli
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/url"
+	"time"
+
+	"github.com/dcos/dcos-cli/pkg/config"
+	"github.com/dcos/dcos-cli/pkg/httpclient"
+	"github.com/dcos/dcos-cli/pkg/login"
+	"github.com/dcos/dcos-cli/pkg/mesos"
+	"github.com/dcos/dcos-cli/pkg/prompt"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
+	"github.com/spf13/pflag"
+)
+
+// SetupFlags are command-line flags for a cluster setup.
+type SetupFlags struct {
+	caBundle     []byte
+	caBundlePath string
+	noCheck      bool
+	insecure     bool
+	loginFlags   *login.Flags
+	fs           afero.Fs
+}
+
+// NewSetupFlags creates flags for a cluster setup.
+func NewSetupFlags(fs afero.Fs, loginFlags *login.Flags) *SetupFlags {
+	return &SetupFlags{
+		loginFlags: loginFlags,
+		fs:         fs,
+	}
+}
+
+// Register registers cluster setup flags.
+func (f *SetupFlags) Register(flags *pflag.FlagSet) {
+	flags.BoolVar(
+		&f.insecure,
+		"insecure",
+		false,
+		"Allow requests to bypass SSL certificate verification (insecure).",
+	)
+	flags.BoolVar(
+		&f.noCheck,
+		"no-check",
+		false,
+		"Do not check CA certficate downloaded from cluster (insecure). Applies to Enterprise DC/OS only.",
+	)
+	flags.StringVar(
+		&f.caBundlePath,
+		"ca-certs",
+		"",
+		"Specify the path to a list of trusted CAs to verify requests against.",
```

Comment:
What does it mean a `path to a list`. Is it a directory which all files are accepted. Is it a path to a single file?

Reply:
Good point, it is a path to a single file, which might contain a single or multiple certificate authorities.

Reply:
Changing to `Specify the path to a file with trusted CAs to verify requests against.`